### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by DTRademaker**
@gcroci2 
Simple starting pipeline. Here, each node tries to separately predict if the peptide is binding or not, the final decision is based on the sum of the predictions. Special here is also that weight_decay was needed to keep the GNN stable. For now I only did random selection of the validation set, not the clusters (although I added the cluster-ID under 'score' to the hdf5 file so it is possible, please download the newest version of the dataset from the snellius server).
To improve the model I think we should look into input-feature representation/normalization. Besides the NAN and inf values we could change the representations (log mean/std manipulation, for example). The features are very inconsistent, some have one-hot encoding while others are still using integers. After that we should check which features make the model better/worse. 
These hdf5 are now also only using residue level nodes, which means 9 nodes per peptide. We should also built the atom based hdf5 files.

### CODE REVIEW

**Review by svenvanderburg — CHANGES_REQUESTED** [review]
Great work @DTRademaker! This is an important milestone 🎉 

I only looked at the code from a meta-level. I have some questions/remarks:
1. I might be wrong, but it seems like you have rewritten deeprank-gnn-2. I thought the idea was that we should all use that as a library for training our models. And if it doesn't fit our needs we should adapt it, now this is turning into a deeprank-gnn-3 🙃 Maybe you could submit a PR to deeprank-ggn-2 to add the missing functionality?
2. In (collaborative) ML projects in my opinion it is important to be able to easily replicate and compare experiments and results. I am used to storing in a central place (a spreadsheet will do): the specifics of the approach, the evaluation score, the train and test dataset that were used. What are you used to?
3. Looking at the learning curve, it seems like this model is not learning anything (apart from memorizing the training data). I think it would be good to discuss what can be improved for the next model.

Let's discuss!
